### ISSUE TITLE
configuration issue or a strategy issue?backtest no result.

### ISSUE THREAD

**Issue by DPZZxlz** [freqtrade/freqtrade#10289]
## Describe your environment

  * Operating system: ____windows docker
  * Python Version: _____3.12.3 (`python -V`)
  * CCXT version: _____ 4.3.35(`pip freeze | grep ccxt`)
  * Freqtrade Version: ____freqtrade 2024.5 (`freqtrade -V` or `docker compose run --rm freqtrade -V` for Freqtrade running in docker)
  
## Your question
I've made a lot of attempts, solved str&str, unlimited, StaticPairList with pair_whitelist, re-downloaded futures data (BTC/USDT:USDT), short trading futures market setup... But when I was done with everything, the backtest was inconclusive. I don't know if it's a configuration issue or a strategy issue, I can't fix this.

docker compose run --rm freqtrade backtesting --strategy BollSquze --dry-run-wallet 10000 --config user_data/config.json --timeframe 1h --timerange 20230105-20240105

###result:
====================================================== BACKTESTING REPORT =====================================================
|           Pair |   Entries |   Avg Profit % |   Tot Profit USDT |   Tot Profit % |   Avg Duration |   Win  Draw  Loss  Win% |
|----------------+-----------+----------------+-------------------+----------------+----------------+-------------------------|
|  BTC/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  ETH/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
| LINK/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  WLD/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  XRP/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  SOL/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
| DOGE/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
| AVAX/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  LTC/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  DOT/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  TRX/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|  UNI/USDT:USDT |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
|          TOTAL |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
=============================================== LEFT OPEN TRADES REPORT ===============================================
|   Pair |   Entries |   Avg Profit % |   Tot Profit USDT |   Tot Profit % |   Avg Duration |   Win  Draw  Loss  Win% |
|--------+-----------+----------------+-------------------+----------------+----------------+-------------------------|
|  TOTAL |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
================================================== ENTER TAG STATS ===================================================
|   TAG |   Entries |   Avg Profit % |   Tot Profit USDT |   Tot Profit % |   Avg Duration |   Win  Draw  Loss  Win% |
|-------+-----------+----------------+-------------------+----------------+----------------+-------------------------|
| TOTAL |         0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
==================================================== EXIT REASON STATS =====================================================
|   Exit Reason |   Exits |   Avg Profit % |   Tot Profit USDT |   Tot Profit % |   Avg Duration |   Win  Draw  Loss  Win% |
|---------------+---------+----------------+-------------------+----------------+----------------+-------------------------|
|         TOTAL |       0 |           0.00 |                 0 |           0.00 |           0:00 |     0     0     0     0 |
================================ SUMMARY METRICS =================================
No trades made. Your starting balance was 10000 USDT, and your stake was 500 USDT.

### config.json:
```
    "max_open_trades": 3,
    "stake_currency": "USDT",
    "stake_amount": 500,
    "tradable_balance_ratio": 0.99,
    "fiat_display_currency": "USD",
    "dry_run": true,
    "dry_run_wallet": 10000,
    "cancel_open_orders_on_exit": false,
    "trading_mode": "futures",
    "margin_mode": "isolated",
    "unfilledtimeout": {
        "entry": 10,
        "exit": 10,
        "exit_timeout_count": 0,
        "unit": "minutes"
    },
    "entry_pricing": {
        "price_side": "same",
        "use_order_book": true,
        "order_book_top": 1,
        "price_last_balance": 0.0,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "exit_pricing":{
        "price_side": "same",
        "use_order_book": true,
        "order_book_top": 1
    },
    "exchange": {
        "name": "binance",
        "key": "",
        "secret": "",
        "ccxt_config": {},
        "ccxt_async_config": {},
        "pair_whitelist": ["BTC/USDT:USDT", "ETH/USDT:USDT", "BNB/USDT:USDT","LINK/USDT:USDT","WLD/USDT:USDT","XRP/USDT:USDT","SOL/USDT:USDT","DOGE/USDT:USDT","AVAX/USDT:USDT","LTC/USDT:USDT","DOT/USDT:USDT","TRX/USDT:USDT","UNI/USDT:USDT"
        ],
        "pair_blacklist": [
            "BNB/.*"
        ]
    },
    "pairlists": [
        {
            "method": "StaticPairList",
            "number_assets": 20,
            "sort_key": "quoteVolume",
            "min_value": 0,
            "refresh_period": 1800
        }
    ],
```

###strategy:
```
from pandas import DataFrame
import talib.abstract as ta
import freqtrade.vendor.qtpylib.indicators as qtpylib

from freqtrade.strategy import (
    BooleanParameter,
    CategoricalParameter,
    DecimalParameter,
    IStrategy,
    IntParameter,
)

class BollSquze(IStrategy):

    INTERFACE_VERSION = 3

    lev = 10

    roi = 0.0015

	# ROI table:
    minimal_roi = {
		"0": (roi * lev)
	}

	# Stoploss:
    stoploss = -0.05

	# Trailing stop:
    trailing_stop = False
    trailing_stop_positive = 0.01
    trailing_stop_positive_offset = 0.03
    trailing_only_offset_is_reached = True

	# Sell signal
    use_exit_signal = True
    exit_profit_only = False
    exit_profit_offset = 0.01
    ignore_roi_if_entry_signal = False
    # Optimal timeframe for the strategy.

    timeframe = "1h"
    can_short = True

    bb_squeeze_threshold = IntParameter(30, 110, default=30, space='buy', optimize=True,load=True)

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        # Calculate Bollinger Bands
        bollinger = qtpylib.bollinger_bands(qtpylib.typical_price(dataframe), window=20, stds=2)
        dataframe["bb_lower"] = bollinger["lower"]
        dataframe["bb_middle"] = bollinger["mid"]
        dataframe["bb_upper"] = bollinger["upper"]
 

        # 计算布林带压缩
        dataframe['bb_squeeze'] = ((dataframe['bb_upper'] - dataframe['bb_lower']) / dataframe['bb_middle']) * 100

        return dataframe

    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:

        dataframe.loc[
            ( 
                (dataframe['close'] > dataframe['bb_middle']) 
              #  & (dataframe['bb_squeeze'] > self.bb_squeeze_threshold.value)  # Squeeze is higher than threshold
            ),
            'enter_long'] = 1


        dataframe.loc[
            (
                (dataframe['close'] < dataframe['bb_middle']) 
              #  & (dataframe['bb_squeeze'] > self.bb_squeeze_threshold.value)  # Squeeze is higher than threshold
            ),
            'enter_short'] = 1

        return dataframe

    def populate_exit_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:

        dataframe.loc[
            (
                (dataframe['close'] < dataframe['bb_middle']) 
               # & (dataframe['bb_squeeze'] > self.bb_squeeze_threshold.value)  # Squeeze is higher than threshold
            ),
            'enter_long'] = 1


        dataframe.loc[
            (
                (dataframe['close'] > dataframe['bb_middle']) 
               # &  (dataframe['bb_squeeze'] > self.bb_squeeze_threshold.value)  # Squeeze is higher than threshold
            ),
            'enter_short'] = 1


        return dataframe
```

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

**Comment by stash86** [comment]
You are setting the wrong column in populate_exit_trend, which cause colliding signal issue https://www.freqtrade.io/en/latest/strategy-customization/#colliding-signals
